sycl/test/optional_kernel_features/fp64-conv-emu.cpp

```diff
@@ -0,0 +1,24 @@
+// UNSUPPORTED: cuda, hip
+
+// RUN: %clangxx -O0 -fsycl -fsycl-device-only -fsycl-targets=spir64_gen -fsycl-fp64-conv-emu %s -c -S -emit-llvm -o %t.ll
+// RUN: cat %t.ll
```

Comment:
This intermediate code dump will be removed before merging. Added for testing purpose.